database_apis/users_database.py

```diff
@@ -25,7 +25,9 @@ def __init__(self):
                                 email TEXT,
                                 role TEXT CHECK( role IN ('admin', 'general', 'investigator') ) NOT NULL DEFAULT 'general',
                                 name TEXT,
-                                password TEXT
+                                password TEXT,
+                                mac_address TEXT,
+                                date TEXT
```

Comment:
Added new column to the user table: mac_address of user's computer and date - the time when the user enables auto login, taking the format of xxxx-xx-xx xx:xx:xx  (year-month-day hour-minute-second).